Tests/SonarScanner.MSBuild.PreProcessor.Tests/SonarWebServiceTest.cs

```diff
@@ -99,6 +99,28 @@ public void LogWSOnError()
             }
         }
 
+        [TestMethod]
+        public void WarnIfDeprecated_SQ79_ShouldNotWarn()
+        {
+            this.ws = new SonarWebService(this.downloader, "http://myhost:222", this.logger);
+            this.downloader.Pages["http://myhost:222/api/server/version"] = "7.9.0.2232";
```

Comment:
Let's use `[DataTestMethod]` with more versions like 6.7, 7.0 and the existing 7.9